src/widgets/infinite-hits/infinite-hits.tsx

```diff
@@ -34,92 +34,84 @@ export type InfiniteHitsCSSClasses = {
   /**
    * The root element of the widget.
    */
-  root: string | string[];
+  root?: string | string[];
+
   /**
    * The root container without results.
    */
-  emptyRoot: string | string[];
+  emptyRoot?: string | string[];
+
   /**
    * The list of results.
    */
-  list: string | string[];
+  list?: string | string[];
+
   /**
    * The list item.
    */
-  item: string | string[];
+  item?: string | string[];
+
   /**
    * The “Show previous” button.
    */
-  loadPrevious: string | string[];
+  loadPrevious?: string | string[];
+
   /**
    * The disabled “Show previous” button.
    */
-  disabledLoadPrevious: string | string[];
+  disabledLoadPrevious?: string | string[];
+
   /**
    * The “Show more” button.
    */
-  loadMore: string | string[];
+  loadMore?: string | string[];
+
   /**
    * The disabled “Show more” button.
    */
-  disabledLoadMore: string | string[];
+  disabledLoadMore?: string | string[];
 };
 
 export type InfiniteHitsTemplates = {
   /**
    * The template to use when there are no results.
    */
-  empty: Template<{ results: SearchResults }>;
+  empty?: Template<{ results: SearchResults }>;
+
   /**
    * The template to use for the “Show previous” label.
    */
-  showPreviousText: Template;
+  showPreviousText?: Template;
+
   /**
    * The template to use for the “Show more” label.
    */
-  showMoreText: Template;
-  /**
-   * The template to use for each result.
-   */
-  item: Template<Hit>;
-};
+  showMoreText?: Template;
 
-export type InfiniteHitsRendererWidgetParams = {
   /**
-   * Escapes HTML entities from hits string values.
-   *
-   * @default `true`
-   */
-  escapeHTML?: boolean;
-  /**
-   * Enable the button to load previous results.
-   *
-   * @default `false`
-   */
-  showPrevious?: boolean;
-  /**
-   * Receives the items, and is called before displaying them.
-   * Useful for mapping over the items to transform, and remove or reorder them.
+   * The template to use for each result.
    */
-  transformItems?: (items: any[]) => any[];
+  item?: Template<Hit>;
```

Comment:
Is this AlgoliaHit or Hit? The Hit also contains __position.

Reply:
Hit

Reply:
See answer above. From reading the code, that's the internal hit—but you can try it in a test or sandbox.